graphql/src/main/kotlin/com/trib3/graphql/execution/LeakyCauldronHooks.kt

```diff
@@ -258,7 +291,15 @@ class DateTimeHooks : SchemaGeneratorHooks {
             LocalDate::class -> LOCAL_DATE_SCALAR
             LocalTime::class -> LOCAL_TIME_SCALAR
             OffsetDateTime::class -> OFFSET_DATETIME_SCALAR
+            UUID::class -> UUID_SCALAR
             else -> null
         }
     }
 }
+
+// add deprecated alias for backwards compatibility
+@Deprecated(
+    "DateTimeHooks has been renamed LeakyCauldronHooks",
+    ReplaceWith("LeakyCauldronHooks")
+)
+typealias DateTimeHooks = LeakyCauldronHooks
```

Comment:
hmm this doesn't work, will try moving it to a different file and see if that works

Reply:
it works in a separate file, but our builds fail w/ deprecated anyway, so just going to remove as we need a version bumpt anyway